docs/howto/how-to-contribute.md

```diff
@@ -1 +1,72 @@
 # How to Contribute
+
+_ONE_ always welcomes your contribution, but there are basic guidelines that you should follow in
+order to make your contribution be accepted.
+
+This document explains such guidelines for beginners.
+
+## General contribution guidelines
+
+If you are not familiar with git or github, please visit
+[here](https://guides.github.com/activities/hello-world/) for basic understanding of git and github.
+
+## How to create a Pull Request
+
+This section explains the steps to create a pull request (PR).
+
+1. Create an issue
+
+   Maintainers will accept your contribution only when it is well aligned with the 
+   [roadmap](./overview/roadmap.md) and design principles of _ONE_. So, it is optional, but 
+   recommended for contributors to create an issue and have a discussion with maintainers before 
+   writing code.
+
+1. Create a draft PR
+
+   Maintainers will accept your pull request only when it is **reasonably small** and **focused**.
+   Sometimes, your contribution may require huge and loosely-coupled changes. You **should** split
+   your contribution into multiple small, but focused pull requests in this case. Unfortunately, it
+   is possible that maintainers reject your pull request as it is hard for them to understand the
+   intuition behind these changes. So, it is optional, but recommended for contributors to present
+   the full [draft](https://github.com/Samsung/ONE/pulls?q=is%3Apr+is%3Aopen+label%3ADRAFT) of your 
```

Comment:
(optional) for showing also closed drafts how about using `https://github.com/Samsung/ONE/pulls?q=is%3Apr+label%3ADRAFT+` ?

Reply:
@seanshpark , good point. I will update the PR. :)